draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -758,6 +758,34 @@ any stream which references the failing certificate in question or process the
 requests as unauthenticated and provide error information at the HTTP semantic
 layer.
 
+# Required Domain Certificate Extension {#extension}
+
+The Required Domain extension allows certificates to limit their use with
+Secondary Certificate Authentication.  A client MUST verify that the server has
+proven ownership of the indicated identity before accepting the limited
+certificate over Secondary Certificate Authentication.
+
+The identity in this extension is a restriction asserted by the requester of the
+certificate and is not verified by the CA.  Conforming CAs SHOULD mark the
+requiredDomain extension as non-critical.  Conforming CAs MUST require the
+presence of a CAA record {{!RFC6844}} prior to issuing a certificate with this
```

Comment:
The design in signed exchanges also requires a CAA parameter for the issuance to be allowed.  Because we are interested in limiting the scope of misissuance, perhaps it would be good to define a new CAA parameter and require that the name match the value of the parameter.  That doesn't have to happen here, but we should discuss whether that helps any.